<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>2D Graphics Modeler</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>621</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="MainTab">
    <attribute name="title">
     <string>2D Graphics Render</string>
    </attribute>
    <widget class="QComboBox" name="idComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>540</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="idLabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>520</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Shape ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="moveLabel">
     <property name="geometry">
      <rect>
       <x>219</x>
       <y>520</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Move Shape</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="xSpinBox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>540</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="ySpinBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>540</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="xLabel">
     <property name="geometry">
      <rect>
       <x>181</x>
       <y>542</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="yLabel">
     <property name="geometry">
      <rect>
       <x>251</x>
       <y>542</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newShapeButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>538</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>New Shape</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteShapeButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>538</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Shape</string>
     </property>
    </widget>
    <widget class="RenderArea" name="renderArea" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>1000</width>
       <height>500</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1000</width>
       <height>500</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
        <widget class="QComboBox" name="sortComboBox">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>540</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Area</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Perimeter</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="sortLabel">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>520</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Sort Shape List By:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ReportsTab">
    <attribute name="title">
     <string>Reports</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="ReviewTab">
    <attribute name="title">
     <string>Reviews</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="ContactTab">
    <attribute name="title">
     <string>Contact Us</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header>renderarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
